### PR TITLE
refactor ledger grpc client

### PR DESCRIPTION
**Pull request by laipogo**

### CODE REVIEW

File: protos/ledger/ledger.proto [hunk 1]
```diff
@@ -20,7 +20,7 @@ import "github.com/tron-us/protobuf/gogoproto/gogo.proto";
 
 // Interface exported by the server.
 
-service Channels {
+service ChannelsService {
```
Comment (taiyangc) [inline protos/ledger/ledger.proto]: Should not change this - ledger server is not controlled by BTFS.

File: ledger/ledger.go [hunk 2]
```diff
@@ -19,21 +18,17 @@ import (
...
-		credential := credentials.NewTLS(&tls.Config{RootCAs: cp})
+		credential := credentials.NewTLS(&tls.Config{})
 		grpcOption = grpc.WithTransportCredentials(credential)
 	}
-	conn, err := grpc.Dial(ledgerAddr, grpcOption)
+	conn, err := grpc.Dial(ledgerAddr, grpcOption, grpc.WithBlock())
 	if err != nil {
 		return nil, err
 	}
+	fmt.Println("state:", conn.GetState())
```
Comment (taiyangc) [inline ledger/ledger.go]: remove print

File: ledger/ledger.go [hunk 3]
```diff
@@ -19,21 +18,17 @@ import (
 
 func LedgerConnection(ledgerAddr, certFile string) (*grpc.ClientConn, error) {
```
Comment (taiyangc) [inline ledger/ledger.go]: Why can't you change this to the same as Escrow, Guard, Status client builder?

Whoever's using it (Escrow + go-btfs) will adjust anyway

Comment (laipogo) [reply ledger/ledger.go]: > Why can't you change this to the same as Escrow, Guard, Status client builder?
> 
> Whoever's using it (Escrow + go-btfs) will adjust anyway

working on it, just test that we don't need certFile, will change in next pr.

Comment (laipogo) [reply ledger/ledger.go]: > > Why can't you change this to the same as Escrow, Guard, Status client builder?
> > Whoever's using it (Escrow + go-btfs) will adjust anyway
> 
> working on it, just test that we don't need certFile, will change in next pr.

oh, i see, we can just remove this func, we have LedgerClient in another ledger.go

**Pull request merged**